### PR TITLE
Show warning when action is overwritten by plugin

### PR DESCRIPTION
**Pull request by KrauseFx**
screenshot 2016-05-31 20 20 23

### CODE REVIEW

File: fastlane/lib/fastlane/plugins/plugin_manager.rb [hunk 1]
```diff
@@ -326,10 +336,22 @@ def store_plugin_reference(gem_name)
 
         File.basename(path).gsub("_action", "").gsub(".rb", "").to_sym # the _action is optional
       end
+      references.keep_if { |a| !a.nil? }
```
Comment (mfurtak) [inline fastlane/lib/fastlane/plugins/plugin_manager.rb]: `references.compact!` should do this for you.

Comment (KrauseFx) [inline fastlane/lib/fastlane/plugins/plugin_manager.rb]: Ah, so much Ruby magic, perfect

**Comment by KrauseFx** [comment]
Updated the PR, everything else is good @mfurtak?

**Comment by mfurtak** [comment]
Nice 👍

**Pull request merged**